cl/search/signals.py

```diff
@@ -130,3 +137,26 @@
     AudioDocument,
     oa_field_mapping,
 )
+
+
+@receiver(post_save, sender=RECAPDocument)
```

Comment:
It's recommended to use a `dispatch_uid` in the receiver to ensure that the handler is not registered more than once for the same signal and sender.

```suggestion
@receiver(post_save, sender=RECAPDocument, dispatch_uid="handle_recap_doc_change_uid",)
```

Reply:
Oh ok, cool